openmetadata-ui/src/main/resources/ui/src/components/common/FilterPattern/FilterPattern.tsx

```diff
@@ -42,41 +48,77 @@ const FilterPattern = ({
   };
 
   return (
-    <div className="tw-mt-4" data-testid="filter-pattern-container">
-      <div className="tw-flex tw-items-center">
-        <input
-          checked={checked}
-          className="tw-mr-3 custom-checkbox"
-          data-testid={`${type}-filter-pattern-checkbox`}
-          id={`${type}FilterPatternCheckbox`}
-          name={`${type}FilterPatternCheckbox`}
-          type="checkbox"
-          onChange={(e) => handleChecked(e.target.checked)}
-        />
-
-        <label htmlFor={`${type}FilterPatternCheckbox`}>{`${capitalize(
-          type
-        )} Filter Pattern`}</label>
-      </div>
+    <div className="m-t-md" data-testid="filter-pattern-container">
+      <Row>
+        <Col>
+          <Checkbox
+            checked={checked}
+            className="m-r-sm filter-pattern-checkbox"
+            data-testid={`${type}-filter-pattern-checkbox`}
+            id={`${type}FilterPatternCheckbox`}
+            name={`${type}FilterPatternCheckbox`}
+            onChange={(e) => handleChecked(e.target.checked)}
+          />
+        </Col>
+        <Col className="flex flex-col">
+          <label htmlFor={`${type}FilterPatternCheckbox`}>{`${capitalize(
+            type
+          )} Filter Pattern`}</label>
+          <Typography.Text
+            className="text-grey-muted m-t-xss"
+            data-testid="filter-pattern-info">
+            {t('message.filter-pattern-info', {
+              filterPattern: type,
+            })}{' '}
+            <Typography.Link
+              href={getFilterPatternDocsLinks(type)}
+              target="_blank">
+              {t('label.read-more')}{' '}
+              <SVGIcons
+                alt="external-link"
+                className="m-l-xss"
+                icon={Icons.EXTERNAL_LINK}
+                width="14px"
+              />
+            </Typography.Link>
+          </Typography.Text>
+        </Col>
+      </Row>
       {checked && (
         <div data-testid="field-container">
           <Field>
-            <label className="tw-block tw-form-label">Include:</label>
-            <input
-              className="tw-form-inputs tw-relative tw-form-inputs-padding tw-py-2"
+            <label className="flex flex-col">{t('label.include')}:</label>
+            <Typography.Text
+              className="text-grey-muted m-t-xss m-b-xss"
+              data-testid="filter-pattern-include-info">
+              {t('message.filter-pattern-include-exclude-info', {
+                activity: toLower(t('label.include')),
```

Comment:
we can create a separate field for include-lower instead of using additional method